SConstruct

```diff
@@ -202,7 +202,8 @@ env = Environment(
     "#third_party/acados/include/hpipm/include",
     "#third_party/catch2/include",
     "#third_party/libyuv/include",
-    "#third_party/json11",
+    "#third_party/nlohmann",
+    "#third_party/tobiaslocker",
```

Comment:
we're very careful about adding new dependencies. can you find some other way to do this?

Reply:
I can implement base64 decoding without an external library.

Reply:
Done.